kie-remote/src/main/java/org/kie/remote/CommonConfig.java

```diff
@@ -41,7 +41,7 @@
 
     private static Properties config;
 
-    public static Properties getStatic() {
+    public static synchronized Properties getStatic() {
```

Comment:
Why this change? Is this method invoked by multiple thread concurrently?

Reply:
@danielezonca 
For some reason one our bug hunting plugin fails on this because it is not thread safe for some reasons. Try it with -Dfull parameter. Build fails if method is not "synchronized".

Reply:
ok